qutebrowser/commands/command.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, *, handler, name, instance=None, maxsplit=None,
         if scope != 'global' and instance is None:
             raise ValueError("Setting scope without setting instance makes "
                              "no sense!")
+        # pylint: enable=too-many-locals
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.